lib/routes/gcores/category.ts

```diff
@@ -77,7 +77,10 @@ async function handler(ctx) {
                 const itemPage = itemRes.data;
                 const $ = load(itemPage);
 
-                let articleData = await got(`https://www.gcores.com/gapi/v1${item.url}?include=media`);
+                let articleData = await got(`https://www.gcores.com/gapi/v1${item.url}?include=media,category,user`);
+                // assign value before replace
+                const articleMeta = articleData.data.included[0];
```

Comment:
Consider using something like `.find((i) => i.type === 'user' && i.id === articleData.data.data.relationships.user.data.id)` instead of `[0]` since the order in `included` should be not considered as fixed.

Reply:
Understandable, I'll do that. By the way, I notice that many commits are replacing `got` with `ofetch` recently, is it a preferred/required transition?

Reply:
It depends on you. If you take a look at `utils/got`, you will notice that it becomes a wrapper of [ofetch](https://github.com/unjs/ofetch) instead of [got](https://github.com/sindresorhus/got).